app/models/prueba.rb

```diff
@@ -0,0 +1,2 @@
+class Prueba < ApplicationRecord
```

Comment:
Once you add rules on your migrations, you should also add validations here.
You can take a look at:
https://guides.rubyonrails.org/active_record_validations.html